Base/usr/share/man/man1/beep.md

```diff
@@ -24,6 +25,8 @@ will fail to use the PC speaker.
 $ beep
 # Use beep with tone of 1000Hz
 $ beep -f 1000
+# Use beep with tone of 1000Hz for 1 second
+$ beep -f 1000 -n 1000
```

Comment:
If we keep the positional arguments, both the freshly added example as well as the preexisting one are outdated.